### PR TITLE
Update unifi and unifi base OS versions

### PR DESCRIPTION
**Pull request by thatsamguy**
* Unifi from `5.12.72` to `6.0.43`
* Base OS from Ubuntu `16.04` to `20.04`
* Mongodb from `3.4` to `3.6` & switch to OS supplied version
* enabled https for version compare

### CODE REVIEW

**Comment by Firefishy** [comment]
Very nice. Worked well for me. Upgrade was seemless from existing mongo 3.4 data.

**Comment by Firefishy** [comment]
Unifi `6.0.28` is out: https://community.ui.com/releases/UniFi-Network-Controller-6-0-28/23c77a99-0957-449a-b3c0-58a37d4df81f

**Comment by mykter** [comment]
thanks, works for me. 6.0.36 out now too.
For anyone else who is about to look up how to fetch a pull request like I just did:
```sh
git fetch origin pull/549/head:thatsamguy-unifi
git switch thatsamguy-unifi
```

File: unifi/Dockerfile [hunk 1]
```diff
@@ -64,31 +64,28 @@ RUN set -ex; \
...
 	openjdk-8-jre-headless \
+	logrotate \
+	libcap2 \
 	--no-install-recommends \
 	&& rm -rf /var/lib/apt/lists/*
 
 # unifi version
 # From: https://www.ubnt.com/download/unifi/
-ENV UNIFI_VERSION "5.12.72"
+ENV UNIFI_VERSION "6.0.45"
```
Comment (mykter) [inline unifi/Dockerfile]: now at `6.1.71`

**Comment by Firefishy** [comment]
Unifi 6.1.17 confirmed working for me.

**Comment by Firefishy** [comment]
Unifi 6.2.26 is now out.
gosu can also be moved to Ubuntu based package.

**Comment by Firefishy** [comment]
[Unifi 6.4.54 is now out](https://community.ui.com/releases/UniFi-Network-Application-6-4-54/c1be3b7f-44c4-4d6f-af1e-707bf017110d).

Just bumping `ENV UNIFI_VERSION "6.4.54"` worked fine for me.

**Comment by thatsamguy** [comment]
Closing PR as I'm no longer maintaining this for my own systems.

**Pull request closed without merging**